components/data-services.qmd

```diff
@@ -0,0 +1,23 @@
+---
+title: Data Services
+subtitle: Providing standard tooling, Application Programming Interfaces (APIs), and access to persistent, discoverable Earth science datasets
+---
+
+## Data Services
+
+The goal of VEDA Data Services is to support open science through community standard tooling, Application Programming Interfaces (APIs), and providing access to persistent, discoverable Earth science datasets. This approach includes:
+
+Hosting and making available key datasets in analysis-optimized formats
+Tying together VEDA products by providing access to hosted data, existing third-party geospatial datasets, and new datasets produced within the VEDA ecosystem
+Making data Findable, Accessible, Interoperable, and Reusable (FAIR), within and also beyond VEDA
+The Data Services hosts a suite of services that make these pointers possible.
```

Comment:
These lines all render as one block for me, could we break it up with bullet points or other formatting?
```suggestion
- Hosting and making available key datasets in analysis-optimized formats
- Tying together VEDA products by providing access to hosted data, existing third-party geospatial datasets, and new datasets produced within the VEDA ecosystem
- Making data Findable, Accessible, Interoperable, and Reusable (FAIR), within and also beyond VEDA

The Data Services hosts a suite of services that make these pointers possible.
```
![Screenshot 2024-10-08 at 15-17-43 Data Services – VEDA Documentation](https://github.com/user-attachments/assets/5bfe7579-5ceb-45be-bf9b-cf9496d5b177)